sensor/kubernetes/complianceoperator/utils.go

```diff
@@ -45,7 +45,7 @@ func convertCentralRequestToScanSettingBinding(namespace string, request *centra
 	for _, profile := range request.GetProfiles() {
 		profileRefs = append(profileRefs, v1alpha1.NamedObjectReference{
 			Name:     profile,
-			Kind:     complianceoperator.ScanSetting.Kind,
```

Comment:
Interesting - this must have been causing an issue during integration tests?

Reply:
Yes that was causing an error when writing or running the objects.  If memory serves it could not find the profile until I updated this.

Reply:
Got it - yeah that makes sense. Thanks!